### PR TITLE
trim acknowledgements and legends

### PR DESCRIPTION
**Pull request by jharenza**
### Purpose

Work towards #432

### CODE REVIEW

File: content/03.results.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ To ensure that new code executed in the development environment, we used the con
...
... diffuse midline glioma, Other HGG = other high-grade glioma, ATRT = atypical teratoid rhabdoid tumor, MB = medulloblastoma, Other ET = other embryonal tumor, EPN = ependymoma, PNF = plexiform neurofibroma, DNET = dysembryoplastic neuroepithelial tumor, CRANIO = craniopharyngioma, EWS = Ewing sarcoma, CPP = choroid plexus papilloma). C, Overview of the open analysis and manuscript contribution model. A contributor proposed an analysis, implemented it in their fork, and filed a pull request (PR) to add changes to the analysis repository. PRs underwent review for scientific rigor and implementation correctness. Using container and continuous integration technologies, PRs were checked to ensure all software dependencies were included and code was not sensitive to underlying data changes. Finally, a contributor filed a PR documenting their methods and results to the Manubot-powered manuscript repository for review. D, A potential path for an analytical PR. Arrows indicate revisions. Panel A created with BioRender.com.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/2eb889d2d495a7327fb51e377df2f9d780c47117/figures/pngs/figure1.png?sanitize=true){#fig:Fig1 width="7in"}
```
Comment (jaclyn-taroni) [inline content/03.results.md:28]: I wonder if the figure legends get double-counted in our word count. You may have already performed testing to check, but if not, we should.

**Review by jaclyn-taroni — APPROVED** [review]
👍🏻  one question about whether or not we've checked to make sure we are not double-counting the figure legends towards our word limit.

**Comment by jaclyn-taroni** [comment]
I'm going to merge this, as I don't think the question about word count blocks merging.

**Comment by jharenza** [comment]
> 👍🏻  one question about whether or not we've checked to make sure we are not double-counting the figure legends towards our word limit.

Nope- I will remove the in text ones during my word counting so that I can get results counts without them, then for legends, I was counting main figures only from the section at the end. Unclear if tables count but those two are minimal text

**Pull request merged**